src/quantile.rs

```diff
@@ -8,7 +8,7 @@ use {MaybeNan, MaybeNanExt, Sort1dExt};
 pub mod interpolate {
     use ndarray::azip;
     use ndarray::prelude::*;
-    use num_traits::{FromPrimitive, ToPrimitive, NumOps};
+    use num_traits::{FromPrimitive, NumOps, ToPrimitive};
```

Comment:
This is probably due to `rustfmt`, I can remove if it is un-necessary.

Reply:
No worries, I am working on getting `rustfmt` in the CI pipeline :+1: (see #31)

Reply:
#31 has been merged, this should be ok now once you merge master back into your branch :+1: